state-sync/data-streaming-service/src/metrics.rs

```diff
@@ -141,6 +141,15 @@ pub static PENDING_DATA_RESPONSES: Lazy<IntGauge> = Lazy::new(|| {
     .unwrap()
 });
 
+/// Counter for the number of complete pending data responses
+pub static COMPLETE_PENDING_DATA_RESPONSES: Lazy<IntGauge> = Lazy::new(|| {
+    register_int_gauge!(
+        "aptos_data_streaming_service_complete_pending_data_responses",
+        "Counters related to the number of complete pending data responses",
```

Comment:
"Counters related to" is an odd prefix? would this be correct as just "The number of complete pending data responses"
(See also PENDING_DATA_RESPONSES "Counters related to")

Reply:
Yeah, this pattern is used all over the place. I'm gonna leave it as-is for now. We can clean it up if we wish, but I'd rather do that separately (because it'll be a big PR 😄).

Reply:
The wording might make sense for histogram_vec, but even there it should probably be more specific :P